ecs-init/volumes/ecs_volume_plugin.go

```diff
@@ -125,6 +165,12 @@ func (a *AmazonECSVolumePlugin) Create(r *volume.CreateRequest) error {
 
 	// record the volume information
 	a.volumes[r.Name] = vol
+
+	// save the state of new volume
+	err = a.state.recordVolume(r.Name, vol)
```

Comment:
this simplifies to return a.state.recordVolume(...)

Reply:
yeah, in the other PR, I log something if err != nil, hence this path 
(https://github.com/aws/amazon-ecs-init/pull/283/files#diff-c11b0b2ff52442451c3df2b3f2380e61R185)